cpp/daal/src/services/compiler/generic/env_detect_features.cpp

```diff
@@ -1,6 +1,7 @@
 /* file: env_detect_features.cpp */
```

Comment:
I would actually suggest to split this particular file into `env_detct_x86_features.cpp` and ``env_detect_arm_features.cpp` in order to not create the `#ifdef` hell